src/modules/location/map/index.ts

```diff
@@ -0,0 +1 @@
+export { Map } from './map';
```

Comment:
you have to create `components` folder in the `location` and then move map to the components